_posts/2019-06-28-dropping-support-for-mysql.md

```diff
@@ -0,0 +1,47 @@
+
+We have dicussed preferring PostgreSQL many times, but [the last time was different](https://community.theforeman.org/t/minimum-rdbm-requirements/14127) and we came to conclusion it's the right thing to do. [The plan](https://community.theforeman.org/t/rfc-dropping-support-for-mysql/14185) was then set and we are at the beginning of the journey where the first and really difficult step is on you, our users:
+
+**Foreman 1.23 will be the last version with MySQL support.**
+
+The upcoming version 1.23 will also feature a warning UI strip in case it's running on MySQL. Don't worry, we will let you close the warning as soon as you see it, but we want to communicate this properly and in advance. We highly recommend to do the database migration to PostgreSQL during upgrade to version 1.23 because if there is a blocker bug that prevents from migrating to MySQL, there is still a backup plan of holding off migration and staying on MySQL for one more release until we come up with version 1.24 which will be PostgreSQL only. This is important:
+
+**Do not wait with the migration for Foreman 1.24.**
+
+Along with the 1.23 release, we will provide official migration documentation, and mention it in our Release Notes as well. If you want to give the migration a test shot today, there's an existing [migration blog post](https://theforeman.org/2017/10/migrating-foreman-from-mysql-to-postgresql.html) written by Timo. We don't expect the official document be very much different. If you do such testing, please come back to us with time estimation so we can tell our users in advance what to expect.
+
+It was not easy decision, we tried hard in the past to support both databases. If you search for [PRs containing MySQL](https://github.com/theforeman/foreman/pulls?page=3&q=is%3Apr+mysql) you'll find counless workarounds, fixes and endless discussions about how to solve problems in a way that doesn't break for PostgreSQL users. There is one common theme everywhere: most developers do not use MySQL both for development and for production testing. This stems from the fact that Red Hat, a company which pays most of the users to do day-to-day development of Foreman, only have a product that is based on PostgreSQL. So we often try to reproduce things on PostgreSQL or blindly provide patches and let affected users to hotfix their instances. Also the biggest and most popular plugin called Katello does not support MySQL, so many community users are not using it at all. Although we haven't asked about it explicitly in the Foreman Community Survey yet, we believe that the majority of our users use PostgreSQL already.
```

Comment:
Red Hat isn't the reason developers don't use MySQL, some RH developers actually do, and in any case this isn't the reason for moving away from PG. I would drop that sentence.

Reply:
Sure removing.